ops/iiif.tmpl.yaml

```diff
@@ -32,6 +32,9 @@ extraVolumeMounts: &volMounts
   - name: uploads
```

Comment:
this file seems unused?

Reply:
There's seems to be a valid host: hyku-iiif.$BASE_URL

and it correlates to a running rancher pod: 

![image](https://github.com/samvera/hyku/assets/10081604/7ac1b3e6-0f05-4d98-b0f0-4e2df4da574c)